py/torch_tensorrt/dynamo/lowering/passes/draw_fx_graph.py

```diff
@@ -0,0 +1,53 @@
+import torch
+from torch.fx import passes
+from torch_tensorrt.dynamo._settings import CompilationSettings
+from torch_tensorrt.dynamo.lowering.passes import LoweringPassSignature
+
+PRE_DEBUG_NAME = {
+    0: "exported_program",
+    1: "after_remove_detach,",
+}
+
+POST_DEBUG_NAME = {
```

Comment:
we need to find a programmatic way to generate these names, this will be hard to maintain